src/read-documents/read-documents.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import { UsersService } from 'src/users/users.service';
+import { ReadDocuments } from './models';
+
+@Injectable()
+export class ReadDocumentsService {
+  constructor(
+    @InjectModel(ReadDocuments)
+    private readDocuments: typeof ReadDocuments,
+    private usersService: UsersService
+  ) {}
+
+  async findOneUser(userId: string) {
+    const user = await this.usersService.findOne(userId);
+    return user;
```

Comment:
Ici tu peux juste faire `return this.usersService.findOne(userId);` (mon IDE met des warning sur ça)